commands/settings/log_lock_waits.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+const settings = require('../../lib/setter')
+
+function explain (setting) {
+  if (setting.value) {
+    return `When a deadlock is detected, a log message will be emitted in your application's logs.`
+  }
+  return `When a deadlock is detected, no log message will be emitted in your application's logs.`
+}
+
+module.exports = {
+  topic: 'pg',
+  command: 'settings:log_lock_waits',
```

Comment:
we try to avoid it if all possible, but yeah, kebab-case is what the style guide specifies